canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.scss

```diff
@@ -55,3 +57,12 @@
 		}
 	}
 }
+
+.properties-dropdown-label {
+	display: flex;
+	justify-content: space-between;
+	.custom-icon {
+		padding-left: 160px;
```

Comment:
Where did you get this number?  There should be a way to place the icon at the end of that is what is desired.